utils_cv/detection/dataset.py

```diff
@@ -96,38 +150,64 @@ def __init__(
             self.train_ds,
             batch_size=self.batch_size,
             shuffle=True,
-            num_workers=4,
+            num_workers=db_num_workers(),
```

Comment:
For windows setting num workers to zero is much faster due to known issue in pytorch